floodmodeller_api/units/logical_rules.py

```diff
@@ -0,0 +1,76 @@
+"""
+Flood Modeller Python API
+Copyright (C) 2023 Jacobs U.K. Limited
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License 
+as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty 
+of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details. 
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see https://www.gnu.org/licenses/.
+
+If you have any query about this program or this License, please contact us at support@floodmodeller.com or write to the following 
+address: Jacobs UK Limited, Flood Modeller, Cottons Centre, Cottons Lane, London, SE1 2QG, United Kingdom.
+"""
+
+import pandas as pd
+
+from .helpers import join_10_char, join_n_char_ljust
+from ..diff import check_item_with_dataframe_equal
+
+### Variables, Rules & Varrules Classes ###
+
+
+class Variables:
+    """Class to hold VARIABLES"""
+
+    def __init__(self, var_block):
+        self._read(var_block)
+
+    def __repr__(self):
+        return f"<floodmodeller_api Variables Class: Variables()>"
+
+    def _read(self, var_block):
+        header = [
+            "name",
+            "type",
+            "initial value",
+            "initial status",
+        ]
+        data_list = []
+        for line in var_block[1:-1]:
```

Comment:
It is better to keep the variable names more dexcriptive if possible, e.g.`name`, `type`, `initial_value`